copier/main.py

```diff
@@ -243,7 +269,7 @@ def _cleanup(self) -> None:
         for method in self._cleanup_hooks:
             method()
 
-    def _check_unsafe(self, mode: Literal["copy", "update"]) -> None:
+    def _check_unsafe(self, mode: Operation) -> None:
```

Comment:
I'm fine with either, just wanted to point out that in my mind there is a minor semantical difference:

`operation` is always the user-requested, high-level operation.
`mode` in this function can be `copy` during an update since `run_copy` is called inside `run_update`, which in turn calls `_check_unsafe` with `copy`.

Reply:
Makes sense, let's keep `mode`.